notebook/static/base/js/security.js

```diff
@@ -59,7 +57,7 @@ define([
          * sanitize just the css in style tags in a block of html
          * called by sanitize_html, if allow_css is true
          */
-        var h = $("<div/>").append(html);
+        var h = document.createElement('div').append(html);
```

Comment:
I don't think this works. element.append only adds a single node, not hierarchical html text.